docs/how-to-work-on-coding-challenges.md

```diff
@@ -7,9 +7,9 @@
 
 Our goal is to develop a fun and clear interactive learning experience.
 
-Designing interactive coding challenges is difficult. It would be much easier to write a lengthy explanation or to create a video tutorial, and there's a place for those on Medium and YouTube. However, for our core curriculum, we're sticking with what works best for most people - a fully interactive, video game-like experience.
+Designing interactive coding challenges is difficult. It would be much easier to write a lengthy explanation or to create a video tutorial, and there's a place for those on Medium, YouTube and many other platforms. However, for our core curriculum, we're sticking with what works best for most people - a fully interactive, video game-like experience.
```

Comment:
```suggestion
Designing interactive coding challenges is difficult. It would be much easier to write a lengthy explanation or to create a video tutorial, and there's a place for those on our Developer News and our YouTube channel. However, for our core curriculum, we're sticking with what works best for most people - a fully interactive, video game-like experience.
```